.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
```suggestion
      interval: "monthly"
```

Reply:
@matthiasbeyer, I've added suggestions, so you can just push the btn "Commit suggestion" and then merge without the necessity to be waiting for the author.